pyomo/contrib/pynumero/interfaces/ampl_nlp.py

```diff
@@ -702,14 +711,18 @@ def ineq_constraint_names(self):
         (corresponding to evaluate_ineq_constraints)"""
         return list(self._con_ineq_idx_to_name)
 
+    @deprecated(msg='This method has been replaced with primal_idx', version='', remove_in='6.0')
```

Comment:
Save comment as for `variable_names()` above.

Reply:
done.